services/dynamic-sidecar/Dockerfile

```diff
@@ -117,12 +117,12 @@ COPY --chown=scu:scu --from=prod-only-deps  ${DY_VOLUMES} ${DY_VOLUMES}
 COPY --chown=scu:scu services/dynamic-sidecar/docker services/dynamic-sidecar/docker
 RUN chmod +x services/dynamic-sidecar/docker/*.sh
 
-# more agressive checking helps with detecting the searvice is ready
-HEALTHCHECK --interval=30s \
-  --timeout=1s \
-  --start-period=5s \
-  --retries=3 \
-  CMD ["python3", "services/dynamic-sidecar/docker/healthcheck.py", "http://localhost:8000/health"]
+# improoving boot times by disabling the healtcheck
+# in oder to have similar performance a more aggressive healethcek
+# would be required.
+# since the director-v2 already keeps track of this service
+# removing the healthchek is not an issue
+HEALTHCHECK NONE
```

Comment:
how much performance increase do you get by doing this? 5 seconds? if that the case I really do not see a need of removing it.

Reply:
I remember up to 30 seconds of speedup.

Reply:
ok. if you are absolutely sure that the dyn sidecar is going to handle it...

Reply:
Please notice there is no docker-compose file for this service. The `dynamic-sidecar` is a bit special. It gets started on the fly by the director-v2 when it needs to run the new version of a dynamic service.

The `director-v2` already health checks all spawned dynamic-sidecars continuously. I think no further enforcing of a health check is required since it brings no advantage in this case.

Before a service is marked as ready by docker and if a health check, said health check must pass. It is very hard to align this healtcheck to pass as soon as the dynamic-sidecar boots; it depends on a combination of various factors: such as CPU speed & availability, network.

Reply:
- True, i missed that it does not use a docker-compose. I wonder if it can be also setup via CLI but it is not so straightforward anymore ...
- Yes, i am aware that the ``director-v2``manages the lifetime of the dynamic-sidecar. OK, on second thought, that might cover most of it

ok.